app/helpers/reactjs_helper.rb

```diff
@@ -11,58 +10,60 @@ def react_component(name, props = {})
     content_tag('foreman-react-component', '', :name => name, :data => { props: props })
   end
 
-  def webpacked_plugins_with_global_css
-    global_css_tags(global_plugins_list).join.html_safe
+  def js_tags_for(requested_plugins)
+    requested_plugins.map do |plugin|
+      javascript_tag("window.tfm.tools.loadPluginModule('/webpack/#{plugin.to_s.tr('-', '_')}','#{plugin.to_s.tr('-', '_')}','./index');".html_safe)
```

Comment:
In `global_js_tags` you use `plugin[:id]` and here just `plugin`. That should be consistent.